support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -205,8 +200,9 @@ function buildRegularPaymentRequest(
 		deliveryInstructions,
 		csrUsername,
 		salesforceCaseId,
-		threeTierCreateSupporterPlusSubscription,
 		debugInfo: actionHistory,
+		threeTierCreateSupporterPlusSubscription:
```

Comment:
Presume this creates the double payment at the backend as labelled `threeTierCreateSupporterPlusSubscription:true` ?